MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -827,12 +830,11 @@ public string EscapeSubtitleFilterPath(string path)
             return path.Replace('\\', '/').Replace(":", "\\:").Replace("'", "'\\\\\\''");
         }
 
-        /// <summary>
-        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
-        /// </summary>
+        /// <inheritdoc />
         public void Dispose()
         {
             Dispose(true);
+            GC.SuppressFinalize(this);
```

Comment:
why?

Reply:
Standard .Net dispose pattern

Reply:
got a link to read about this?

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose?redirectedfrom=MSDN